wagtail/admin/templates/wagtailadmin/home/locked_pages.html

```diff
@@ -49,7 +49,12 @@ <h2 class="title-wrapper">{% trans "Your locked pages" %}</h2>
                                 </ul>
                             </td>
                             <td>{# Deliberately empty #}</td>
-                            <td valign="top"><div class="human-readable-date" title="{{ page.locked_at|date:"DATETIME_FORMAT" }}">{% blocktrans trimmed with time_period=page.locked_at|timesince_simple %}{{ time_period }}{% endblocktrans %}</div></td>
+                            <td valign="top">
+                                <button type="button" class="button-tooltip" data-tippy-content="{{ page.locked_at|date:"DATETIME_FORMAT" }}">
+                                    <time datetime="{{ page.locked_at|date:"c" }}">
+                                        {% blocktrans trimmed with time_period=page.locked_at|timesince_simple %}{{ time_period }}{% endblocktrans %}
+                                    </time>
+                                </button>
                         </tr>
```

Comment:
I think the closing `td` is missing here

Reply:
Good catch, I've fixed it